SuperBuild/External_python-dicom-requirements.cmake

```diff
@@ -38,7 +11,7 @@ endif()
 ExternalProject_Include_Dependencies(${proj} PROJECT_VAR proj DEPENDS_VAR ${proj}_DEPENDENCIES)
 
 if(Slicer_USE_SYSTEM_${proj})
-  foreach(module_name IN ITEMS pydicom numpy six certifi idna chardet urllib3 requests)
+  foreach(module_name IN ITEMS pydicom numpy six)
```

Comment:
certifi, idna, chardet, etc. are grouped together for `requests` in `python-requests-requirements`. That project is listed as a dependency for python-dicom-requirements here.  I can remove numpy here though. Six is uniquely needed for various unrelated projects, but I didn't feel the need to pull it out to its own individual project. It could be though so not to ever have different version of Six specified.